README.md

```diff
@@ -1,88 +1,127 @@
+####As an agent who is logged in you will:
 
-- New trip requests (a users pending trips)
-- Total income generated this year (will be 10% of user
-  trip cost)
-- Travelers on trips for today's date
+- See any pending trips
+- Be able to approve any pending trips
+- See any trips happening on today's date
+- Be able to search for a traveler and then approve or delete any trips for that traveler
 
-#### Traveler Interaction
+#####To approve a pending trip:
+Locate the trip you would like to approve in the PENDING TRIPS section
+Click APPROVE
+You will see a SUCCESS message
+The trip will no longer be in the PENDING TRIPS section upon refreshing and logging back in as agent OR logging out and logging back in as agent
+#####To search for a traveler and approve / delete pending trip:
+Search for the traveler by name in the search field
+Click SHOW TRAVELER
+Any PENDING trips will be shown first in the list of this travelers trips
+To APPROVE the trip, click APPROVE
+To DELETE the trip, click DELETE
+Depending on the choice above the trip will now be showing as either APPROVED in the travelers list of UPCOMING TRIPS or will no longer be showing at all if it was DELETED
 
-**A traveler who is logged in is able to make a trip request:**
+#### To login as any one of 50 travelers use the following login information:
 
-- Select a date, duration, number of travelers
-- Choose from a list of destinations
-  After making these selections, they see an estimated
-  cost (including a 10% travel agent fee) for the trip.
-- Once they submit the trip request, it will show on
-  their dashboard as “pending” so that the travel agency
-  can approve or deny it.
+    Username: traveler1  ( 1 represents the TravelerID and can be any number from 1 - 50 )
+    Password: travel2020
 
-#### Agent Interaction
+#### As a traveler who is logged in you will:
 
-**An agent who is logged in is able to:**
+- See your upcoming trips
+- See your past trips
+- Be able to make a trip request
 
-- See and approve / deny trip requests
-- Search for any user by name
-- View their:
-  _ name
-  _ a list of all of their trips
-  _ The total amount they’ve spent (including
-  10% agent cut)
-  _ Approve a trip request for that user - Delete an
-  upcoming trip for that user
+#####To create a trip request:
+Fill in the appropriate fields (destination, number of travelers, start date, and end date) and click CALCULATE
+The trip information will then be displayed on the screen. Click CONFIRM to send the post request to the API.
+The requested trip will be shown in your PENDING TRIPS section the next time you login.
+Refresh the page and log back in OR click LOGOUT and then log back as the same traveler to see the new trip in the PENDING TRIPS section
 
-### Setup
+# Screenshot
 
-**Git Setup**
+![Initial Styling of project](https://i.imgur.com/RSSlSew.png)
 
-```
-$ git clone git@github.com:josh-everett01/travel_tracker.git
+# Setup
```

Comment:
setup, then use triple ticks for a code block

```
$ bash command one
$ bash command two
$ bash command three
```

This section should come before all your functionality overview